.github/scripts/prLabelSemver.js

```diff
@@ -0,0 +1,43 @@
+// prLabelSemver.js
+const { prData } = require("./prData");
+const appendSummary = require("./actionUtils");
+const fs = require("fs");
+
+/**
+ * Automatically labels pull requests based on semantic versioning (semver) guidelines
+ * and appends a summary to the GitHub action step.
+ *
+ * @param {object} github - The github object providing context and operations for the pull request.
+ * @param {object} context - The context object containing metadata and states for the action run.
+ * @param {object} core - The core library with utilities for logging and error handling.
+ */
+const prLabelSemver = async ({ github, context, core }) => {
```

Comment:
You're still using lambda syntax here, and the return value should be a promise.